tests/common.bash

```diff
@@ -13,6 +13,15 @@ function helm() {
 	command helm --kube-context $CLUSTER_CONTEXT "$@"
 }
 
+function helm_in {
+    helm upgrade --install --wait --namespace $NAMESPACE --create-namespace \
+        "${@:2}" $1 $KUBEWARDEN_CHARTS_LOCATION/$1
+
+    # kubewarden-defaults ignore wait param
+    [ $1 = 'kubewarden-defaults' ] && retry "kubectl --context $CLUSTER_CONTEXT rollout status -n kubewarden deployment/policy-server-default"
```

Comment:
Can we call the already existing `wait_for_default_policy_server_rollout()` function where we call `helm_in` for `kubewarden-defaults` instead of doing this?

Reply:
Problem is that helm ignores `--wait` param in kubewarden-defaults.
If I use `wait_for_default_policy_server_rollout` it fails because `deployment/policy-server-default` does not exist.
`retry` function is workaround for this problem - but it relies on `timeout` - which requires command (kubectl...), not a function (wait_for_default-...) as parameter.

I agree it's not pretty, but not sure how to fix it.

Reply:
uook, thanks for the explanation. Is ok as it is :).